octobot_trading/enums.py

```diff
@@ -307,6 +307,7 @@ class WebsocketFeeds(Enum):
     PORTFOLIO = 'portfolio'
     POSITION = 'position'
     TRADE = 'trade'
+    USER_DATA = 'user_data'
```

Comment:
What is user data? There is no channel for this

Reply:
Maybe it's not the right place to put it, it's the stream containing all user-related feeds (balance, orders, etc) in binance ws https://github.com/binance-exchange/binance-official-api-docs/blob/master/user-data-stream.md

Reply:
To match the way `WebsocketFeeds` is used in other WS. Prefer using : 
`BALANCE = 'user_data'`
`ORDERS = 'user_data'`